openslides_backend/action/actions/meeting/clone.py

```diff
@@ -67,6 +67,13 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
                 value = instance.pop(field)
                 self.get_meeting_from_json(meeting_json)[field] = value
 
+        for mediafile_id in instance["meeting"]["mediafile"]:
```

Comment:
Could you use get("mediafile", [])?

Reply:
Yes,  I could. But why? The meeting comes from the export_meeting function and must have a mediafile, or?
Okay, changed.